modin/pandas/test/dataframe/test_default.py

```diff
@@ -20,6 +20,7 @@
 from numpy.testing import assert_array_equal
 import io
 
+from modin.pandas.test.test_groupby import modin_df_almost_equals_pandas
```

Comment:
maybe it's more logical to move `modin_df_almost_equals_pandas` to the `test/utils.py` and then import it in `test_groupby` and `test_default`?

Reply:
I agree, moved.